charts/verdaccio/templates/deployment.yaml

```diff
@@ -67,7 +67,7 @@ spec:
             - mountPath: /verdaccio/storage
               name: storage
               readOnly: false
-            {{- if .Values.useSecretHtpasswd }}
+            {{- if .Values.secrets.htpasswd }}
```

Comment:
if someone is already using this, to me, looks as a breaking change.

Reply:
yes, flag is deprecatedd, and version bumped to 4.3.0 
Thanks for the catch